lhotse/kaldi.py

```diff
@@ -207,26 +209,30 @@ def export_to_kaldi(
             )
         )
 
-    # Create a simple CutSet that ties together the recording <-> supervision information.
+    # Create a simple CutSet that ties together
+    # the recording <-> supervision information.
     cuts = CutSet.from_manifests(
         recordings=recordings, supervisions=supervisions
     ).trim_to_supervisions()
 
     # wav.scp
     save_kaldi_text_mapping(
         data={
-            recording.id: make_wavscp_string(
+            f"{recording.id}_{channel}": make_wavscp_channel_string_map(
```

Comment:
I didn’t notice that before but this is possibly breaking existing workflows where data is exported to Kaldi for processing and then imported back to Lhotse (recording ids are changing). I am aware of such use cases. Could you avoid modifying recording ID for single channel recordings?

Reply:
will do. Will you take "if" instead of making the list comprehension even more complex? :)

Reply:
Yeah. I think at some point this module might need unit tests, it’s definitely growing in complexity. But don’t worry about it for now.